### PR TITLE
Publish maven artifacts via travis

### PR DESCRIPTION
**Pull request by sscdotopen**
*Issue #13:*

Integration for automatically publishing tagged artifacts to maven central.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,9 +4,50 @@
...
 
-    <groupId>com.amazon</groupId>
+    <groupId>com.amazon.deequ</groupId>
     <artifactId>deequ</artifactId>
-    <version>1.1</version>
+    <version>1.0-SNAPSHOT</version>
+
+    <name>deequ</name>
+    <description>Deequ is a library built on top of Apache Spark for defining "unit tests for data",
+        which measure data quality in large datasets.
```
Comment (andrey-tpt) [inline pom.xml]: measure(s) ?

File: .travis.yml [hunk 2]
```diff
@@ -1 +1,16 @@
 language: java
+jdk: oraclejdk8
+
+env:
+  global:
+    - secure: "AE6KjEwnhkzjrMH3kz1+xoD0dCNhHoMmLF3kzb/EDDgR+slOacqnTkC+xC73d3c6UTWx3JLjxEXu9GYOoSgwb1kbR7eTzUHEJ5ADsIhCRPQIlu1qevA3xfsGUz09IN1x2a8DKDYiHT1WuYFtTwfGcvTPOuSExnjlxcUDvt8ydYKn3Fd0jHGzAnAowT+yErkVnsvlTWjglGwXCTPDIj9rzWyhOFXm5qNYIrzLkWnRL1l29u6Vti0y4msAsQ+sGF+GLbIfmmNw1aoDvS1vh+8uNR2HibdCW94HNQQ5GCqcOCKVpHSBSyWiAJ6XLO4uWkA95p1qktG7MYTgksq7I3I0EuYI1uAyZkRiY1g0b9xveQOrw7LFVB7LlBqN578xuUCIM3PputZ3sIfZg4RQsd5VGFu+u5ujF9al4ynzXjHOFD0yOFsPC47Xvfpydta9m6B2nETOsVIVjtDETO7s3r9AbUZbmWkgEkX1oMqzL5nUjyikIf1k++BuMwv75FbBOCKEXX0d/1JovCRSCi9JZUjSg6XfR+XKD7mIqNTzEJV6vUJZWqy6shfdFR8xAsD5iSNAQZEB9TyCspK8CgFPPjlor2Va8eHWkg912Ai6gEZQm/LJLBLEu63GuLFXi0bk0GKEQ8Vz8omOka/iT1B3jlm9jWaW240MQ7TnsdWNnwtPSNU="
```
[the remark this answers is not in the document]
Comment (sscdotopen) [reply .travis.yml]: will do

**Comment by sscdotopen** [comment]
We will squash the commits once the CR is approved and we merge it.

File: pom.xml (hunk 1 above)
Comment (sscdotopen) [reply pom.xml]: tests measure :)

**Pull request merged**